src/WebJobs.Script/Metrics/MetricConstants.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class MetricConstants
```

Comment:
nit: these can be used for more than just metrics. Recommend the following:

1. Move attributes (tags) to another file `TelemetryAttributes`.
2. For metrics meter and instrument names, we can either have this file (`MetricConstants`), or we can have the metric implementation itself have these constants (so move them into `HostMetricsMeter`, which would be renamed `HostMetrics` by my other suggestion).
3. With them moved to `HostMetrics`, can rename `AzureFunctionsMeter` to just `MeterName`.